### ISSUE TITLE
403 When creating group call with service hosted media

### ISSUE THREAD

**Issue by Alexp2019** [microsoftgraph/microsoft-graph-comms-samples#64]
When attempting to create a group call with service hosted media, I receive a 403: Request authorization tenant mismatch.
Creating a regular call works fine. 
Running a POST to the following URL:
https://graph.microsoft.com/beta/app/calls
I've essentially given all app permissions available to my test tenant to rule that out.
JWT Perms:
"roles": [
    "Chat.UpdatePolicyViolation.All",
    "Calls.JoinGroupCall.All",
    "OnlineMeetings.Read.All",
    "OnlineMeetings.ReadWrite.All",
    "User.ReadWrite.All",
    "Application.ReadWrite.OwnedBy",
    "People.Read.All",
    "Application.ReadWrite.All",
    "Calls.InitiateGroupCall.All",
    "Group.Read.All",
    "Calls.JoinGroupCallAsGuest.All",
    "Sites.Read.All",
    "Sites.ReadWrite.All",
    "Group.ReadWrite.All",
    "Sites.Manage.All",
    "User.Invite.All",
    "User.Read.All",
    "Chat.Read.All",
    "ChannelMessage.Read.All",
    "User.Export.All",
    "Chat.ReadWrite.All",
    "ChannelMessage.UpdatePolicyViolation.All",
    "Member.Read.Hidden",
    "Calls.AccessMedia.All",
    "Sites.FullControl.All",
    "Calls.Initiate.All"
  ],

Body:
 ```
{
      "subject": "Test Call123",
      "callbackUri": "https://url.ngrok.io/callback/calling",
      "source": {
      "identity": {
          "application": {
          "id": "appID"
          }
        }
      },  
      "targets": [
      {
          "identity": {
          "user": {
              "id": "userID",
              "displayName": "test name"
          }
        }
        },
        {
        "identity": {
          "user": {
              "id": "userID2",
              "displayName": "test name2"
          }
          }
      }   
      ],
      "requestedModalities": [ "audio" ],
      "mediaConfig": {
        "@odata.type": "#microsoft.graph.serviceHostedMediaConfig",
        "preFetchMedia": [
          {
            "uri": "https://cdn.contoso.com/beep.wav",
            "resourceId": "1D6DE2D4-CD51-4309-8DAA-70768651088E"
          },
          {
            "uri": "https://cdn.contoso.com/cool.wav",
            "resourceId": "1D6DE2D4-CD51-4309-8DAA-70768651088F"
          }
        ]
      },
      "chatInfo": {
        "threadId": "19:meeting_aergaergOTdlMjY0YTFiNTA0@thread.v2",
        "messageId": "0",
        "replyChainMessageId": null
      }
    
    }

```

**Comment by ksikorsk** [comment]
Hello,

Unfortunately placing a call to multiple targets is not yet supported.  To work around this you first need to create an ad-hoc meeting, join the meeting with the bot, then invite your participants.

`POST /app/onlineMeetings`
with one of your participants as the organizer (bot cannot be organizer).  Keep note of the [ChatInfo](https://microsoftgraph.github.io/microsoft-graph-comms-samples/docs/core_calls/Microsoft.Graph.OnlineMeeting.html?q=OnlineMeeting#Microsoft_Graph_OnlineMeeting_ChatInfo) and [Organizer](https://microsoftgraph.github.io/microsoft-graph-comms-samples/docs/core_calls/Microsoft.Graph.OnlineMeeting.html?q=OnlineMeeting#Microsoft_Graph_OnlineMeeting_Participants).

`POST /app/calls`
with the [ChatInfo](https://microsoftgraph.github.io/microsoft-graph-comms-samples/docs/core_calls/Microsoft.Graph.Call.html?q=Call#Microsoft_Graph_Call_ChatInfo) from above and the [OrganizerMeetingInfo](https://microsoftgraph.github.io/microsoft-graph-comms-samples/docs/core_calls/Microsoft.Graph.Call.html?q=Call#Microsoft_Graph_Call_MeetingInfo) with the organizer from above.

`POST /app/calls/{id}/participants/invite`
Once the call is created you need to [invite](https://docs.microsoft.com/en-us/graph/api/participant-invite?view=graph-rest-beta&tabs=javascript) the target participants

**Comment by ksikorsk** [comment]
@VinodRavichandran FYI: This is a request for placing a call to multiple participants:
- Create ad-hoc call
- Invite participants

**Comment by VinodRavichandran** [comment]
@Alexp2019  We are working on enabling creation with multiple participants and will follow that up with inviting multiple participants. A rough timeline for the former will be towards the end of CY19Q3 and subsequent Q for the latter.

**Comment by Alexp2019** [comment]
Hi @ksikorsk , 

Thanks for your assistance. I was able to create a meeting and invite mutliple participants in the same request using that workaround.

One quick question, do you know if it is  possible to invite external users into the meeting? This is the example body being posted to beta/app/calls/callID. 

{
    "participants": [
      {
        "endpointType": "default",
        "identity": {
            "user": {
                "id": "id",
                "displayName": "idname"
            }
      }
    },
    {
        "endpointType": "default",
        "identity": {
            "user": {
                "id": "id",
                "displayName": "test user2"
            }
      }
    }
    ],
    "clientContext": "d45324c1-fcb5-430a-902c-f20af696537c"
  }

```
Thanks

**Comment by ksikorsk** [comment]
@Alexp2019 do you mean users outside of the call tenant?  Federation cases (cross tenant) are currently not supported.

**Comment by Alexp2019** [comment]
@ksikorsk Yep users outside the call tenant. Ok thanks - do you know if it's something that will be available in the future?

**Comment by ksikorsk** [comment]
As far as I know there is no current plan to support federated cases, but @VinodRavichandran can speak more to it.

**Comment by VinodRavichandran** [comment]
It'll be supported when Teams support federation. We don't have a timeline to share publicly.

**Comment by mwerghemmi** [comment]
Hi @VinodRavichandran, 

Is it planned to support calling PSTN number ?

**Comment by VinodRavichandran** [comment]
@mwerghemmi Yes, we are working on that feature, don't have a timeline to share for the entire feature, as its been worked out in phases.

**Comment by bogdan-raduta** [comment]
> @mwerghemmi Yes, we are working on that feature, don't have a timeline to share for the entire feature, as its been worked out in phases.

Hi @VinodRavichandran, 

Do you have any updates regarding support for calling PSTN number?

**Comment by VinodRavichandran** [comment]
@bogdan-raduta  We are planning on making inbound PSTN call support to be available in next Q and follow that up with outbound support.

**Comment by bogdan-raduta** [comment]
Hi @VinodRavichandran,

Any news for the PSTN call support. Do you think we can have the PSTN call support available in Q1 2020, at least in development/test mode?
